main/ESPAudio.cpp

```diff
@@ -680,8 +696,13 @@ void Audio::dactask(void* arg )
 					}
 					dacDisable();
 				}
-				if( deadband_active && amplifier_shutdown.get() )
+				if( shutdownamp ) {
+					if( deadband_active 
+					 || ( audio_mute_sink.get() && _te < 0 )
+					 || ( audio_mute_menu.get() && gflags.inSetup )
+					 || ( audio_mute_gen.get() != AUDIO_ON ) )
 					enableAmplifier( false );
```

Comment:
This repeating of the silencing conditions is not a good implementation.  (And not quite correct regarding general muting during alarms.)  My intention is to submit the RICO audio style version, which also has a better implementation of this detail, using a variable that flags the intention to shut down the amplifier.